From: Carolyn Wyborny <carolyn.wyborny@intel.com>
Date: Tue, 3 Jun 2014 23:50:22 +0000
Subject: [PATCH 107/144] i40e/i40evf: Update RSS configuration
Patch-mainline: v3.17-rc1
Git-commit: e157ea30606743f2bfa2ec445a0bb0839e17271b (partial)
References: bsc#909486 FATE#317393

This patch changes the RSS configuration to set table size
and write to hardware to confirm RSS table size being used.

Change-ID: I455a4c09c9dd479f5791ee1f09fdc83ff9908df5
Signed-off-by: Carolyn Wyborny <carolyn.wyborny@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/intel/i40evf/i40e_type.h |    3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/net/ethernet/intel/i40evf/i40e_type.h
+++ b/drivers/net/ethernet/intel/i40evf/i40e_type.h
@@ -1165,4 +1165,7 @@ enum i40e_reset_type {
 	I40E_RESET_GLOBR	= 2,
 	I40E_RESET_EMPR		= 3,
 };
+
+/* RSS Hash Table Size */
+#define I40E_PFQF_CTL_0_HASHLUTSIZE_512	0x00010000
 #endif /* _I40E_TYPE_H_ */
